src/Compressors/helpers/fwht.jl

```diff
@@ -0,0 +1,108 @@
+"""
+    fwht!  
+
+Performs a Fast Walsh Hadamard Transform (FWHT), modifying the vector `x`. This means that 
```

Comment:
Try:

"Performs an in-place Fast Walsh Hadamard Transform on the vector `x`." and you can remove the next line about copying `x`.